recipe/meta.yaml

```
@@ -35,12 +35,15 @@ outputs:
         - make
         - pkg-config
       host:
+        - cuda-nvml-dev ={{ cuda_compiler_version }}       # [cuda_compiler_version == "12.0"]
```

It looks like UCX is [dynamically linking to `cudart`]( https://github.com/openucx/ucx/blob/69f7bf458218211c775759eb0a021acd371a0f08/config/m4/cuda.m4#L50 ) (as opposed to [using the static `cudart`]( https://github.com/openucx/ucx/blob/69f7bf458218211c775759eb0a021acd371a0f08/config/m4/cuda.m4#L79 ))

So think we need to add `cuda-cudart-dev` to `host`. This will [automatically handle adding the dependency in `run`]( https://github.com/conda-forge/cuda-cudart-feedstock/blob/05f196f348ae9f0161a4080c20586eb7b17e41a4/recipe/meta.yaml#L102-L103 )

```suggestion
        - cuda-cudart-dev ={{ cuda_compiler_version }}     # [cuda_compiler_version == "12.0"]
        - cuda-nvml-dev ={{ cuda_compiler_version }}       # [cuda_compiler_version == "12.0"]
```

---

That's right, pushed the change now. Thanks John.